src/core.c/Nil.pm6

```diff
@@ -1,5 +1,4 @@
 my class Nil is Cool { # declared in BOOTSTRAP
-
```

Comment:
Looks like this is an artifact from the rebase.

Reply:
The empty line removal?  Probably.  Is it a problem?  Don't  think so?

Reply:
Problem, no, just clutters the git history.